pkg/internal/feed/exporter_report.go

```diff
@@ -59,7 +59,6 @@ func (e *ReportExporter) SaveReports(ctx context.Context, apiClient *httpapi.Cli
 	e.Logger.Info("Generating inspection reports")
 
 	status := GetExporterStatus()
-	status.Reset()
```

Comment:
I think we need this when we start the export

Reply:
done.